themes/custom_tmpl/GETTING_STARTED.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
```

Comment:
I see more and more communities putting their documentation in their code. I'd be happy to link from here to the wiki.